docs/openapi/definitions/auth.yaml

```diff
@@ -0,0 +1,217 @@
+openapi: 3.0.1
+info:
+  title: Auth API
+  description:
+    Backstage passport-based auth-provider API. Each provider that is configured
+    in `@backstage/auth-backend` is supplied with the documented endpoints.
```

Comment:
was confused a bit by the formulation that "providers" are "supplied with ... endpoints", wasn't sure what that means